valet-key/ValetKey.Web/WebRole.cs

```diff
@@ -2,21 +2,17 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 namespace ValetKey.Web
 {
+    using Azure.Storage.Blobs;
     using Microsoft.Azure;
     using Microsoft.WindowsAzure.ServiceRuntime;
-    using Microsoft.WindowsAzure.Storage;
 
     public class WebRole : RoleEntryPoint
     {
         private const string BlobContainer = "valetkeysample";
```

Comment:
Let's refactor this name to a project variable and reuse in the controllers too.